docker/README.md

```diff
@@ -5,15 +5,20 @@
 
 ## How to deploy
 
-It is as simple as:
+Prerequisites:
+A postgres database with semver extension is needed. Using abrt/postgres_w_semver
+image is recommended.
+`docker run --name faf_db -dit abrt/postgres_w_semver`
 
-`docker run --name faf -dit abrt/faf-image`
+Running FAF is as simple as:
 
-However you also probably want to mount volumes to `/var/lib/postgres` and
-to `/var/spool/faf` not to lose database and FAF's data.
+`docker run --name faf -dit --link faf_db abrt/faf-image`
 
-`docker run --name faf -v /var/lib/faf-docker/faf:/var/spool/faf -v
-/var/lib/faf-docker/postgres:/var/lib/postgres/ -dit abrt/faf-image`
+However you also probably want to mount volumes to `/var/spool/faf` not to lose 
+FAF's data.
+
+`docker run --name faf -v /var/lib/faf-docker/faf:/var/spool/faf --link faf_db
```

Comment:
This wouldn't work because:
- faf_db is missing the POSTGRESQL_ADMIN_PASSWORD variable and user faf is not created.
- faf is missing the PG{USER, PASSWORD, HOST, PORT, DATABASE} variables.

The --link is a legacy option [1] and user-defined networks [2] should be used instead. 
e.g.:
```docker network create faf-network```
```docker run --network faf-network --name faf_db ...```
```docker run --network faf-network --name faf ... ```

[1] https://docs.docker.com/compose/compose-file/#links
[2] https://docs.docker.com/network/